src/DevBetterWeb.Core/ValueObjects/Birthday.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using CSharpFunctionalExtensions;
+
+namespace DevBetterWeb.Core.ValueObjects;
+
+public class Birthday : ValueObject
+{
+  public int Day { get; }
+  public int Month { get; }
+  private const int LEAP_YEAR = 2020;
```

Comment:
Why is this hard-coded to 2020?

Reply:
I wanted to make sure that users could input 29 as a date in February. In order for the DateOnly constructor to not throw, the year has to be a leap year. 2020 is a leap year. I'm only using the DateOnly as an easy way to validate day/month combos and format them. I actually just store the day and month as ints.

If you are wondering why use const, I dunno, because it's a constant. Isn't that what const is for? It's my Typescript that is bleeding into my C#. 😁